README.md

```diff
@@ -1,102 +1,269 @@
 # druid
 
-## Data-oriented Rust User Interface Design toolkit
+## A data first Rust-native UI toolkit.
 
-Druid is a new Rust-native UI toolkit, still in early stages. Its main
-goal is performance, also aiming for small binary size and compile time,
-fast startup, and very easy build configuration (just `cargo run`).
-It currently supports Windows, macOS, and GNU/Linux.
+[![crates.io](https://meritbadge.herokuapp.com/druid)](https://crates.io/crates/druid)
+[![docs.rs](https://docs.rs/druid/badge.svg)](https://docs.rs/druid/)
+[![license](https://img.shields.io/crates/l/druid)](./LICENSE)
 
-Raph gave a talk at the July 2018 SF Rust Meetup ([video][jul-2018-video],
-[slides][jul-2018-slides]) about the design. Traditional object-oriented
-designs have proved to be cumbersome.
+Druid is an experimental Rust-native UI toolkit. Its main goal is to demonstrate
+the potential of Rust as an application programming language, while letting
+users write fast, small, and generally efficient programs with minimal hassle.
```

Comment:
Not really required but would be nice to explicitly mention that druid is under active development and its API might change.

Sorry if I am late for the party :)

Reply:
Would it be helpful if we used a word like "alpha" or something for this? Not sure how to phrase it:
The druid toolkit is currently in an alpha state and its API is not set in stone.

Reply:
That will work for me at least.

Reply:
It says "experimental" in the first line, I don't think adding the word "alpha" adds much. (This is a subtle point, but "alpha" implies that it's intended to be a product, that there will be a beta and a 1.0, which we're not promising).

How about this: We have been doing periodic releases of druid on crates.io, but it is under active development and its API might change.

Reply:
Great.